server/lib/chat/chat-api.ts

```diff
@@ -326,6 +318,28 @@ export class ChatApi {
 
     ctx.status = 204
   }
+
+  @httpGet('/:channelId/info')
```

Comment:
It seems like we don't have a `GET` for `/:channelId`, this should probably just be that?

Reply:
When I make this into just a `/:channelId` route, then all the requests to `/batch-info` get intercepted by this route :/
Maybe that one should be renamed to something else too?

Reply:
`batch-info` needs to come before the `/:channelId` route so that the router hits it first. In general you should always order your routes from constants -> tokens

Reply:
You could probably also do an explicit match for `channelId`'s format if you want to be extra safe, I think like:

`/:channelId(\\d+)`